{"name": "Тест входа в профиль курьера с неверным логином или паролем", "status": "passed", "description": "Тест проверяет появление ошибки в профиль курьера с неверным логином(1) и паролем(2)", "parameters": [{"name": "login", "value": "'valid_login'"}, {"name": "password", "value": "'invalidpass'"}, {"name": "expected_status", "value": "404"}, {"name": "expected_message", "value": "'Учетная запись не найдена'"}], "start": 1721217085502, "stop": 1721217085503, "uuid": "6e680b10-fcd0-4e80-be1e-10ba05f1225c", "historyId": "997b73e9d03930d6250bd07dcbcb0e11", "testCaseId": "ec706951e0c7470079769d6d675d4cda", "fullName": "tests.test_login_courier.TestLoginCourier#test_login_invalid_credentials", "labels": [{"name": "feature", "value": "Тест входа в профиль курьера"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-L3DLL49"}, {"name": "thread", "value": "12168-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}